src/gfx/mesh.hpp

```diff
@@ -37,6 +37,8 @@ struct MeshFormat {
   IndexFormat indexFormat = IndexFormat::UInt16;
   std::vector<MeshVertexAttribute> vertexAttributes;
 
+  size_t computeVertexSize() const;
```

Comment:
getVertexSize() is more common

Reply:
Named it compute since it does some extra work than just returning a cached size